Pipfile

```diff
@@ -9,6 +9,7 @@ structlog = "*"
 gunicorn = "*"
 gevent = "*"
 sdc-cryptography = "*"
+google-cloud-pubsub = "==1.7.0"
```

Comment:
Can you try and use * instead.  For most of our repos we do and it works so it should *hopefully* work here